app/assets/stylesheets/spotlight/_blacklight_overrides.scss

```diff
@@ -0,0 +1,16 @@
+.submit-search-text {
+  @extend .sr-only;
+}
+
+.search-form {
+  .input-group-addon {
+    &.for-search-field {
+      border-radius: 4px 0 0 4px;
+    }
+
+    .search_field {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `search_field` should be written in lowercase with hyphens